vnext/ReactUWP/EndPoints/dll/react-native-uwp.arm.def

```diff
@@ -38,6 +38,7 @@ EXPORTS
 ?utf16ToUtf8@unicode@react@facebook@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@ABV?$basic_string@_WU?$char_traits@_W@std@@V?$allocator@_W@2@@5@@Z
 ?utf16ToUtf8@unicode@react@facebook@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@ABV?$basic_string_view@_WU?$char_traits@_W@std@@@5@@Z
 ?utf8ToUtf16@unicode@react@facebook@@YA?AV?$basic_string@_WU?$char_traits@_W@std@@V?$allocator@_W@2@@std@@ABV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z
+?MakeMemoryMappedBuffer@JSI@Microsoft@@YA?AV?$shared_ptr@VBuffer@jsi@facebook@@@std@@QB_WI@Z
```

Comment:
>?Ma [](start = 0, length = 3)

please alphabetize

Reply:
same for other def files

---
In reply to: [321853272](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r321853272) [](ancestors = 321853272)

Reply:
I would like send a separate PR for alphabetizing all the def files.

---
In reply to: [321853374](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r321853374) [](ancestors = 321853374,321853272)